### PR TITLE
[WAF] Update parameters.mdx

### PR DESCRIPTION
**Pull request by luishcardoso**
Hello team,

### Summary

Lets fix the link to custom response settings for blocked requests.

https://developers.cloudflare.com/waf/custom-rules/custom-rulesets/create-dashboard/#configure-a-custom-response-for-blocked-requests

Thanks!

### CODE REVIEW

**Comment by kodster28** [comment]
I think it might be related to the headers further down the page?

https://developers.cloudflare.com/waf/rate-limiting-rules/parameters/#with-response-type-for-block-action

**Comment by luishcardoso** [comment]
> I think it might be related to the headers further down the page?
> 
> https://developers.cloudflare.com/waf/rate-limiting-rules/parameters/#with-response-type-for-block-action

Sure thing.

**Review by luishcardoso — COMMENTED** [review]
Update the links referring to the doc page itself.

File: src/content/docs/waf/rate-limiting-rules/parameters.mdx [hunk 1]
```diff
@@ -113,9 +113,9 @@ Use one of the following values in the API: `block`, `challenge`, `js_challenge`
 
 If you select the *Block* action, you can define a custom response using the following parameters:
 
-* [**With response type**](#with-response-type)
-* [**With response code**](#with-response-code)
-* [**Response body**](#response-body)
+* [**With response type**](/waf/rate-limiting-rules/parameters/#with-response-type-for-block-action)
+* [**With response code**](/waf/rate-limiting-rules/parameters/#with-response-code-for-block-action)
+* [**Response body**](/waf/rate-limiting-rules/parameters/#response-body-for-block-action)
```
Comment (pedrosousa) [inline src/content/docs/waf/rate-limiting-rules/parameters.mdx]:
```suggestion
* [**With response type**](#with-response-type-for-block-action)
* [**With response code**](#with-response-code-for-block-action)
* [**Response body**](#response-body-for-block-action)
```
No need for a path when the target anchors are in the same page.

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**